app/assets/javascripts/fae/form/inputs/_text.js

```diff
@@ -56,6 +57,38 @@ Fae.form.text = {
     $('.js-markdown-editor:not(.mde-enabled)').each(function () {
       var $this = $(this);
 
+      // set up translate button for markdown fields and add helper text
+      var translate_button = $this.siblings( '.js-translate-button' )
```

Comment:
A standard convention when working with jquery is to prepend a `$` sign to any variables that are jquery objects. This helps to differentiate between jquery vs. non-jquery variables at a glance. Let's also use camelCase when working with JS so `translate_button` should become `$translateButton`

Reply:
updated!

Reply:
Though a few I tried ended up breaking stuff.  Next PR, if you still think some vars should be renamed with the $, we can get on a call!